towhee/utils/hub_utils.py

```diff
@@ -234,6 +255,29 @@ def create(self, password: str, repo_class: int) -> None:
         finally:
             self.delete_token(str(token['id']), password)
 
+    def delete(self, password: str):
+        """
+        Delete a repo under current account.
+
+        Args:
+            password (`str`):
+                Current author's password.
+        """
+        token_name = random.randint(0, 10000)
+        r = self.create_token(token_name, password)
+        token = r.json()
+        url = self._root + '/api/v1//repos/' + self._author + '/' + self._repo
+        try:
+            r = requests.delete(url, headers={'Authorization': 'token ' + str(token['sha1'])})
+            r.raise_for_status()
+        except HTTPError as e:
+            if e.response.status_code == 404:
+                print('Repo not found.')
+            else:
+                raise e
+        finally:
+            self.delete_token(str(token['id']), password)
+
```

Comment:
Junjie said we don't want to expose delete to users

Reply:
Is there a specific reason for this @junjiejiangjjj